css/styles.css

```diff
@@ -96,6 +109,73 @@ nav {
   width: 100%;
 }
 
+/* traido de ofm - related to js mobile menu */
+
+.menu-options {
+  margin: 0 !important;
+  display: none;
+}
+
+.fixed {
+  position: fixed;
+  display: block !important;
+  right: 28px;
+  top: 10px;
+}
+
+.mobile-menu {
+  display: none;
+  list-style: none;
+}
+
+.open {
+  display: block;
+  padding-left: 30px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 0%;
+  overflow: hidden;
+  background-color: #396df2;
+  opacity: 0.9;
+  z-index: 5;
+}
+
+.blur {
+  filter: blur(8px);
```

Comment:
- [ ] Kindly make sure to follow the font weight of the menu options. 
- [ ] Kindly make sure to follow the same background colour for the mobile menu. Currently, it's #396df2 but it should be #6070FF according to Figma design.
- [ ] Kindly make sure to give a blur effect using a backdrop to the mobile menu.
- [ ] Kindly make sure to follow the spacing between the menu options as per the Figma design.

<img width="992" alt="Screenshot 2022-10-11 at 11 46 17 PM" src="https://user-images.githubusercontent.com/70269507/195195564-0f27c6a8-9dcd-4c48-b5cf-945c3967514b.png">